### PR TITLE
Issue #658 - Unable to Build on Ubuntu 

### PR DESCRIPTION
**Pull request by dalealleshouse**
Added ubuntu 16.10 as a target and changed the target framework from netcoreapp1.0 to netcoreapp1.1 - it's now building on ubuntu 16.10.

### CODE REVIEW

**Comment by david-driscoll** [comment]
@DustinCampbell this is a wonderful rid problem, I know you love these!

**Comment by DustinCampbell** [comment]
I was waiting until we released .NET Core 1.1 before looking at this. Now that it's here, @dalealleshouse, could you update the PR to the final release of .NET Core 1.1?

**Comment by DustinCampbell** [comment]
I took a look at this today, and I think this PR actually ends up breaking OmniSharp. The problem is that there is an API breaking change introduced [here](https://github.com/dotnet/corefx/pull/10937/commits/a06e6057511ca074a1f525a02bbf1955cd57897c) between System.Reflection.Metadata, 1.4.1 and System.Reflection.Metadata, 1.4.1-beta-24430-01. Unfortunately, Microsoft.NetCore.App, 1.1 references the former version and Roslyn 2.0.0-rc references the latter version.

The CI passes for this change because OmniSharp.exe is moved to netcoreapp1.1, but all of the tests are still on netcoreapp1.0. So, the tests pass but OmniSharp.exe doesn't work.

File: build.json [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-  "DotNetFolder": ".dotnet",
-  "DotNetInstallScriptURL": "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0-preview2/scripts/obtain",
-  "DotNetChannel": "preview",
-  "DotNetVersion": "1.0.0-preview2-003131",
-  "BuildToolsFolder": ".tools",
-  "ArtifactsFolder": "artifacts",
-  "TestProjects": {
-    "OmniSharp.MSBuild.Tests": [
-      "netcoreapp1.0"
+    "UseSystemDotNetPath": "false",
```
Comment (DustinCampbell) [inline build.json]: FWIW, it'd be nice if this change didn't update the whitespace in the files, which messes up the diff.

File: build.json [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+    "DotNetChannel": "preview",
+    "DotNetVersion": "1.0.0-preview2-003131",
+    "BuildToolsFolder": ".tools",
+    "ArtifactsFolder": "artifacts",
+    "TestProjects": {
+        "OmniSharp.MSBuild.Tests": [
+            "netcoreapp1.0"
+        ],
+        "OmniSharp.Roslyn.CSharp.Tests": [
+            "netcoreapp1.0"
```
Comment (JosephWoodward) [inline build.json]: Does that missing type you're referring to happen to be `System.Reflection.Metadata.ISignatureTypeProvider`?

Comment (DustinCampbell) [inline build.json]: That's the one. We're still working on getting Roslyn updated to .NET Core 1.1, which will address the problem.

**Comment by DustinCampbell** [comment]
Closing this in favor of https://github.com/OmniSharp/omnisharp-roslyn/pull/835.

**Pull request closed without merging**